src/toolkitAPI.c

```diff
@@ -3130,16 +3135,43 @@ int DLLEXPORT swmm_openOpening(int nodeID, int idx)
     // Check if Open
     if (swmm_IsOpenFlag() == FALSE)
     {
-        errcode = ERR_API_INPUTNOTOPEN;
+        errcode =  error_getCode(ERR_API_INPUTNOTOPEN);
     }
     // Check if object index is within bounds
     else if (nodeID < 0 || nodeID >= Nobjects[NODE])
     {
-        errcode = ERR_API_OBJECT_INDEX;
+        errcode =  error_getCode(ERR_API_OBJECT_INDEX);
+    }
+    else
+    {
+        // Close the opening
+        errcode = coupling_closeOpening(nodeID, idx);
     }
-
-    // Close the opening
-    errcode = coupling_closeOpening(nodeID, idx);
     return errcode;
 }
 
+int DLLEXPORT swmm_countOpenings(int nodeID, int *num)
+//
+// Input:   nodeID = Index of desired node
+// Return:  API error
+// Purpose: Open an opening.
+{
+    int errcode = 0;
+
+    // Check if Open
+    if (swmm_IsOpenFlag() == FALSE)
+    {
+        errcode =  error_getCode(ERR_API_INPUTNOTOPEN);
+    }
+    // Check if object index is within bounds
+    else if (nodeID < 0 || nodeID >= Nobjects[NODE])
+    {
+        errcode =  error_getCode(ERR_API_OBJECT_INDEX);
+    }
+    else
+    {
+        // Close the opening
```

Comment:
The comment does not fit the action

Reply:
Yes, I agree. Changed in a subsequent commit.